content/en/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm.md

```diff
@@ -531,10 +531,9 @@ Talking to the control-plane node with the appropriate credentials, run:
 
 ```bash
 kubectl drain <node name> --delete-local-data --force --ignore-daemonsets
-kubectl delete node <node name>
 ```
 
-Then, on the node being removed, reset all `kubeadm` installed state:
+Before removing the node, reset all `kubeadm` installed state on the node:
```

Comment:
thanks for the PR @vinu2003 
let's take the opportunity to correct the old wording and change it to:

> Before removing the node, reset the state installed by kubeadm:

please keep the commits squashed to 1.

Reply:
Yes. Done that.